syntax = "proto2";

package tbclient.FrsPage;

import "tbclient/FrsPage/DataReq.proto";

message FrsPageReqIdl {
  optional DataReq data = 1;
}
